models/classes/theme/ThemeServiceAbstract.php

```diff
@@ -215,9 +214,8 @@ protected function getThemeDetailsProviders()
 
     protected function isTaoAsToolEnabled(): bool
     {
-        return $this->getServiceManager()
-            ->getContainer()
-            ->get(FeatureFlagChecker::class)
-            ->isEnabled(FeatureFlagCheckerInterface::FEATURE_FLAG_TAO_AS_A_TOOL);
+        return $this->getServiceManager()->getContainer()
+            ->get(DynamicConfigProviderInterface::class)
+            ->hasConfig(DynamicConfigProviderInterface::PORTAL_URL_CONFIG_NAME);
```

Comment:
```suggestion
            ->hasConfig(DynamicConfigProviderInterface::PLATFORM_URL_CONFIG_NAME);
```
to make name more generic